{"name": "Невозможно авторизоваться с неверными учетными данными", "status": "passed", "start": 1740351736201, "stop": 1740351736819, "uuid": "cbdb663a-5fc4-4548-8118-3ad69a110f90", "historyId": "7cb8840ed5c768596e318cfc3f13414c", "testCaseId": "7cb8840ed5c768596e318cfc3f13414c", "fullName": "tests.test_courier.TestCourier#test_cannot_login_with_wrong_credentials", "labels": [{"name": "feature", "value": "Курьер"}, {"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_courier"}, {"name": "subSuite", "value": "TestCourier"}, {"name": "host", "value": "LAPTOP-QVE1U26H"}, {"name": "thread", "value": "8164-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_courier"}]}